### ISSUE TITLE
Slug needs to be editable

### ISSUE THREAD

**Issue by ChristopherDosin** [ChristopherDosin/Shopware-6-Blog-Plugin#130]
Currently the slug is generated out of the title and you can't edit the slug.
We need to add a feature, so it's possible to edit the slug also after it has been generated.

Keep in mind that we still have to do a validation so the slug is still unique.

**Comment by claymore008** [comment]
Yes, a validation for uniqueness (cross-languages included) is a good idea.

Another nice-to have, would be the possibility to save different teaser pictures for different languages.
